app/src/main/java/org/resolvetosavelives/red/search/PatientSearchScreen.kt

```diff
@@ -34,7 +36,7 @@ class PatientSearchScreen(context: Context, attrs: AttributeSet) : RelativeLayou
   @Inject
   lateinit var controller: PatientSearchScreenController
 
-  private val backButton by bindView<ImageButton>(R.id.patientsearch_back)
+  private val upButton by bindView<ImageButton>(R.id.patientsearch_up)
```

Comment:
We really should call it "Back" everywhere, and not Up, because the terminology is confusing and Google has failed to make this distinction clearly. Not to mention, users don't understand this at all, and expect it to work something-almost-but-not-quite-entirely-like a "Back" button.

Reply:
On a second thought, this is not entirely true. Users get confused by the hardware back button and not the software one. The software back key always takes you a level back/up within the same app whereas hardware back button flies apps around.